### DISCUSSION TITLE
Codemod plugin doesn't install @wdio/codemod/async folder

### DISCUSSION THREAD

**Discussion by jpong-elation** [webdriverio/codemod#58] [Migration Issue] [answered]
Not sure if anyone has come across this or not.. but I'm trying to use the recently created codemod plugin (❤️ to the creator)!! But am running into an issue where the async folder can't be found (aka doesn't exist). Trying to think of where I've gone wrong but can't think of anything. I'm going to try taking a look and post my findings here but just wanted to see if it was a me thing or not.

**Notice that the async folder is missing which deviates from the README**
<img width="762" alt="image" src="https://user-images.githubusercontent.com/91329479/155814487-76a0bef1-cc77-4c9c-a2b4-fe2da95b76e2.png">

**Readme**
<img width="761" alt="image" src="https://user-images.githubusercontent.com/91329479/155814730-577d46a3-5eff-4a99-b85f-f4397c339c70.png">

**Comment by christian-bromann** [comment, marked as answer]
I can't reproduce this:
<img width="875" alt="Screenshot 2022-02-26 at 19 04 31" src="https://user-images.githubusercontent.com/731337/155854199-a2f69fa6-88d9-4433-a3e7-081e2d6dc950.png">

**Reply by jpong-elation** [reply]
Oh I'm an idiot; I'm still using the previous version 😢. Thanks for double checking though! Feel free to delete this discussion honestly lol.
